_data/config.yml

```diff
@@ -1,3 +1,3 @@
-grpc_release_branch: v1.3.2
+grpc_release_branch: v1.3.x
```

Comment:
One thing to point out is that the grpc code that's running as part of our examples is not taken for this repository - most examples are using a package from a package manager. The only part the examples can get broken in the release branch is when someone updates the examples themselves.  And the only time we would be updating the examples directly in the release branch is when they're broken and we want to fix them (otherwise we'd just make the change on upstream/master).

So I'd still argue that using 1.3.x has negligible risk of breaking the examples and it saves us some work because we only need to update the branch when there's a new minor release.